logs/stream/stream.go

```diff
@@ -130,14 +132,37 @@ func handleLogAnalysis(analyzableLogLines []state.LogLine) ([]state.LogLine, []s
 }
 
 func stitchLogLines(readyLogLines []state.LogLine) (analyzableLogLines []state.LogLine) {
-	for _, logLine := range readyLogLines {
-		if logLine.LogLevel != pganalyze_collector.LogLineInformation_UNKNOWN {
+	var linesToAppend []int
+	var linesToAppendLenSum int
+	var b strings.Builder
+	for idx, logLine := range readyLogLines {
+		if logLine.LogLevel == pganalyze_collector.LogLineInformation_UNKNOWN {
+			if len(analyzableLogLines) > 0 {
+				linesToAppend = append(linesToAppend, idx)
+				linesToAppendLenSum = linesToAppendLenSum + len(logLine.Content)
+			}
```

Comment:
Is the "else" here a "should never happen" case? Or an actual "cannot happen" case? (Or are we intentionally dropping these?) Stepping through the logic, I think it's the first/third case, but I'd like to make sure I understand this correctly.

Reply:
We are intentionally dropping these. The situation where we somehow got the tail end of a multi-line statement, we can't actually parse it (because we're missing the primary log line for it). Thus it gets discarded here. I could add a comment to clarify this behavior.